resources/views/orders/orderdetails.blade.php

```diff
@@ -21,51 +21,55 @@
                 <div class="flex flex-col sm:flex-row justify-between gap-10 lg:w-3/5">
                     <div class="flex flex-row gap-2 items-baseline">
                         <div>{{ __('orders/orders.order-date') }}</div>
-                        <div class="p-2 rounded shadow h-11 dark:border dark:border-gray-700">01-04-2024</div>
+                        <div
+                            class="p-2 rounded shadow h-11 dark:border dark:border-gray-700">{{ App\Helpers\DateFormatter::format($order->order_date) }}</div>
                     </div>
                     <div class="flex flex-row gap-2 items-baseline">
                         <div>{{ __('orders/orders.total-price') }}</div>
-                        <div class="p-2 rounded shadow h-11 dark:border dark:border-gray-700">€60,00</div>
+                        <div class="p-2 rounded shadow h-11 dark:border dark:border-gray-700">{{ $totalPrice }}</div>
                     </div>
-                    <!-- Added to test colors, currently hardcoded -->
                     <div class="sm:mt-2.5">
-                        <p class="{{ \App\Enum\DeliveryStatus::delocalised('Geannuleerd') }}">
-                            Geannuleerd
+                        <p class="{{ \App\Enum\DeliveryStatus::delocalised($order->status) }}">
+                            {{ $order->status }}
                         </p>
                     </div>
                 </div>
             </div>
         </div>
 
 
-        @for($i = 0; $i < 3; $i++)
+        @foreach($orderLines as $orderLine)
 
             <div class="m-4">
                 <div
                     class="flex flex-col gap-1 sm:gap-0 pb-1 sm:pb-0 sm:flex-row justify-between items-center sm:pr-6 rounded-xl border dark:text-white dark:border-gray-700">
-                    <img src="https://placehold.co/200" alt="product image"
-                         class="rounded-t-xl sm:rounded-tr-none sm:rounded-l-xl w-full sm:w-fit h-auto">
-                    <div class="">
-                        <a href="#" class="font-bold">
-                            Product titel
+                    <img src="{{ $orderLine->product_image_path }}" alt="Image of {{ $orderLine->product->name }}"
+                         class="rounded-t-xl sm:rounded-tr-none sm:rounded-l-xl w-full sm:w-48 h-auto">
+                    <div class="w-28 text-center">
+                        <a href="{{ route('orders.product', [ 'name' => $orderLine->product->name, 'groupName' => $orderLine->product->group_name]) }}" class="font-bold">
+                            {{ $orderLine->product->name }}
                         </a>
                     </div>
-                    <div>M</div>
-                    <div>€10,00</div>
-                    <div>2 {{ __('orders/orders.amount-items') }}</div>
+                    <div class="w-28 text-center">{{ $orderLine->product_size }}</div>
+                    <div class="w-28 text-center">{{ $orderLine->product_price }}</div>
```

Comment:
Je bedrag heeft geen euro en is met een "."
{{ number_format($product->productSizes->pluck('pivot.price')->min(), 2, __('common.seperator'), '.') }}

Reply:
Ik zie dat mijn copy paste hier fout is gegaan, deze regel zou wel moeten werken.